src/pal/src/misc/miscpalapi.cpp

```diff
@@ -309,16 +262,25 @@ PAL_Random(
         }
         else
         {
-            for( ; i < dwLength; i++)
+            size_t index = 0;
```

Comment:
We have copy of this for the corelib/framework use here: https://github.com/dotnet/corefx/blob/master/src/Native/Unix/System.Native/pal_random.cpp

Could you please update that one as well?

Reply:
Sure

Reply:
Created PR for the change: https://github.com/dotnet/corefx/pull/27601